src/owl/stats/owl_stats.ml

```diff
@@ -175,9 +175,32 @@ let max = Owl_base_stats.max
 let minmax = Owl_base_stats.minmax
 
 
+type histogram = Owl_base_stats.histogram = {
```

Comment:
Can we just `type histogram = Owl_base_stats.histogram` rather than redefining every field since they are already defined in owl_base_stats .... or I missed something so compiler complains?

Reply:
i tried and the compiler complained... maybe there is a way but i did not find it!